pkg/cmd/roachtest/sqlalchemy.go

```diff
@@ -22,7 +22,7 @@ import (
 var sqlAlchemyResultRegex = regexp.MustCompile(`^(?P<test>test.*::.*::[^ \[\]]*(?:\[.*])?) (?P<result>\w+)\s+\[.+]$`)
 var sqlAlchemyReleaseTagRegex = regexp.MustCompile(`^rel_(?P<major>\d+)_(?P<minor>\d+)_(?P<point>\d+)$`)
 
-var supportedSQLAlchemyTag = "rel_1_3_24"
+var supportedSQLAlchemyTag = "rel_1_4_17"
```

Comment:
nice -- should we also pin the version of sqlalchemy-cockroachdb under test in this script?

Reply:
Talked offline, we'll keep it at master for now and monitor how things go with Gord's work on the adapter.